plugins.json

```diff
@@ -18,7 +18,9 @@
       "server": {
         "executables": {
           "linux-amd64": "server/dist/plugin-linux-amd64",
+          "linux-arm64": "server/dist/plugin-linux-arm64",
```

Comment:
Do these binaries really exist in 0.1.2? I can't find them in https://github.com/mattermost/mattermost-plugin-antivirus/releases/tag/v0.1.2.

Reply:
no, i was not sure if need to update this file. do I need to revert this?

Reply:
Please do. I don't think this change is correct.

Reply:
ahh, looking at it again, that is completely wrong. removed my fault